### PR TITLE
Feature/remove image version

### PR DESCRIPTION
**Pull request by ResearchDaniel**
Add backend/frontend functionality to remove generated image versions.
Only non-selected images can be removed to ensure that there is at least one image left.

### CODE REVIEW

**Comment by ResearchDaniel** [comment]
Skärmavbild 2024-08-09 kl  16 59 09

File: frontend/package.json [hunk 1]
```diff
@@ -34,9 +34,11 @@
 	},
 	"type": "module",
 	"dependencies": {
-		"tailwind-merge": "^2.2.2",
+		"@fortawesome/free-solid-svg-icons": "^6.5.2",
```
Comment (Sparkier) [inline frontend/package.json:37]: You could use lucide for this probably.

File: frontend/src/lib/components/ImageComponent.svelte [hunk 2]
```diff
@@ -16,33 +17,35 @@
...
 			const chapter = parseInt(parts[5]);
 			const paragraph = parseInt(parts[7]);
-			fetchUrl = `${API}/api/books/${selectedBook}/chapters/${chapter}/paragraphs/${paragraph}/image/versions`;
+			fetchUrl = `${API}/api/books/${selectedBook}/chapters/${chapter}/paragraphs/${paragraph}`;
 		}
+		return fetchUrl;
+	}
+
+	function getVersionNumber(src: string) {
+		const parts = src.split('/');
```
Comment (Sparkier) [inline frontend/src/lib/components/ImageComponent.svelte]: Can remove.

File: frontend/src/lib/components/editor/ImageVersion.svelte [hunk 3]
```diff
@@ -0,0 +1,68 @@
+<script lang="ts">
+	import Fa from 'svelte-fa';
+	import { createEventDispatcher } from 'svelte';
+	import { faCircleXmark } from '@fortawesome/free-solid-svg-icons';
```
Comment (Sparkier) [inline frontend/src/lib/components/editor/ImageVersion.svelte:4]: Could use this if you want to remove the extra dependency:
https://lucide.dev/icons/circle-x

**Review by Sparkier — APPROVED** [review]
Could remove extra dependency but fine either way.

File: frontend/src/lib/components/editor/ImageVersion.svelte (hunk 3 above)
Comment (ResearchDaniel) [reply frontend/src/lib/components/editor/ImageVersion.svelte]: Nice, fixed this

**Pull request merged**